packages/lesswrong/components/review/ReviewVoteTableRow.tsx

```diff
@@ -193,8 +197,16 @@ const ReviewVoteTableRow = (
                   {v}
               </LWTooltip>
             )}
+            
+          </div>
+          {eligibleToNominate(currentUser) && <div className={classes.yourVote}>
```

Comment:
Allows users to update their vote on /reviewVoting during the Review Phase (this won't update the reviewVoteScore fields, those are deliberately only updated via migration, to be done once at the end of Preliminary Voting and once at end of Final Voting)